frontend/src/components/pages/ReviewPage.tsx

```diff
@@ -16,6 +16,8 @@ import ConfirmationModal from "../translation/ConfirmationModal";
 import {
   REVIEW_PAGE_RETURN_TO_TRANSLATOR_CONFIRMAITON,
   REVIEW_PAGE_RETURN_TO_TRANSLATOR_BUTTON_MESSAGE,
+  REVIEW_PAGE_SUBMIT_TRANSLATION_CONFIRMAITON,
+  SUBMIT_TRANSLATION_BUTTON_MESSAGE,
```

Comment:
Nit: for consistency, maybe `SUBMIT_TRANSLATION_BUTTON_MESSAGE` can be changed to `REVIEW_PAGE_SUBMIT_TRANSLATION_BUTTON_MESSAGE`.